### PR TITLE
Badge pages

### LINKED ISSUE #4318: Page to list users with a given badge
This would be cute!

### PR DESCRIPTION
**Pull request by schradert**
Closes #4318 

- [x] backend: paginated ListBadgeUsers for retrieving badge's user list
- [x] frontend: badge index page
- [ ] frontend: badge user page
- [ ] frontend: tests
- [ ] frontend: storybook
- [ ] frontend: UI/UX improvements

**Backend checklist**
- [x] Formatted my code by running `autoflake --exclude src/proto -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [x] Added tests for any new code or added a regression test if fixing a bug
- [x] All tests pass
- [x] Run the backend locally and it works
- [x] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

**Web frontend checklist**
- [x] Formatted my code with `make format`
- [x] There are no warnings from `make lint`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Comment by schradert** [comment]
@aapeliv I'm thinking that we probably need pagination here for badges with many users (phone verified, strong verification, etc.), what do you think? I can also just not query the database for these badges in #4318 and just show the number of user accounts or something.

**Comment by aapeliv** [comment]
Yes I think that's right. If you are down, pagination would be great!

File: app/web/features/badges/locales/en.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (aapeliv) [inline app/web/features/badges/locales/en.json:1]: I'm not sure how we should go about localizing these. I guess doing this is not a bad idea. Then if we come up with a better way, we can always copy these translations over!

**Review by aapeliv — DISMISSED** [review]
The backend looks great to me! I haven't been able to run the frontend locally (my working directory is a mess with the notify v2 stuff). But happy to merge once it passes all the tests, or we can look for someone from frontend to have a quick look as well!

**Comment by nabramow** [comment]
@schradert are you still working on this or is it okay if someone else takes it over?

**Comment by schradert** [comment]
Feel free to reassign anything assigned to me @nabramow

**Comment by nabramow** [comment]
@aapeliv What if we finish this ticket just as the backend, since Tristan looks like he mostly finished it. Then @bakeiro we can make a separate ticket for the frontend part of this?

**Comment by aapeliv** [comment]
Continued in #5247 (backend) and #5250 (frontend).

**Pull request closed without merging**